bot/discord_ui.py

```
@@ -1,4 +1,5 @@
 import discord
+from .bot_commands import BotConfig # Added import for BotConfig
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)